src/OpenAL/Silk.NET.OpenAL/AL/AL.cs

```diff
@@ -350,11 +349,33 @@ public static AL GetApi(bool soft = false)
             return ret;
         }
 
+        /// <summary>
+        ///     Attempts to load a native OpenAL extension of type <typeparamref name="T" />.
+        /// </summary>
+        /// <param name="ext">
+        ///     The loaded extension.
+        /// </param>
+        /// <typeparam name="T">
+        ///     Type of <see cref="NativeExtension{T}" /> to load.
+        /// </typeparam>
+        /// <returns>
+        ///     <c>True</c> if the extension was loaded, otherwise <c>False</c>.
+        /// </returns>
+        public bool TryGetExtension<T>(out T ext)
+            where T : NativeExtension<AL>
+        {
+            ext = IsExtensionPresent(ExtensionAttribute.GetExtensionAttribute(typeof(T)).Name)
+                ? (T) Activator.CreateInstance(typeof(T), Context)
+                : null;
+            return ext != null;
+        }
+
         /// <summary>
         /// Gets an instance of the API of an extension to the API.
         /// </summary>
         /// <typeparam name="TExtension">The extension type.</typeparam>
         /// <returns>The extension.</returns>
+        [Obsolete("This method has been deprecated and will be removed in Silk.NET 3.0. Please use TryGetExtension instead.")]
```

Comment:
I know I gave you this code but it exceeds 120 characters.
```suggestion
        [Obsolete
        (
            "This method has been deprecated and will be removed in Silk.NET 3.0. " +
            "Please use TryGetExtension instead."
        )]
```